### ISSUE TITLE
replacing our common files

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/fake-api#102]
#28 

right now i want to get rid of these files:
- [ ] https://github.com/GroceriStar/sd/blob/master/src/data/Favorites/favorites.json
- [ ] https://github.com/GroceriStar/sd/blob/master/src/data/UserGrocery/userGrocery.json
- [ ] https://github.com/GroceriStar/sd/blob/master/src/data/Items/items.json

when you'll be able to recreate them - we'll move this functionality into sd-wrapper and remove this files

**Comment by atherdon** [comment]
when we'll done it - it will close: https://github.com/GroceriStar/groceristar-fetch/issues/192

**Comment by atherdon** [comment]
for the last item at this list, can you also add to sd file with data from https://github.com/GroceriStar/groceristar-fetch/blob/master/src/projects/ChickenKyiv/chickenKyiv.js#L53-L63

**Comment by atherdon** [comment]
i think something is not clear. I see 3 files at sd, but their names are: 
```
    'users',
    'grocery',
    'items'
```
So favorites is not finished

**Comment by vadim9999** [comment]
Sorry, Now I added regenerated in folder output in commit https://github.com/GroceriStar/sd/pull/91/commits/249256759c197e4f253f3d777d8be2eec47486b6

**Comment by atherdon** [comment]
No prob. looks awesome - so MenuGenerator is our last one, right?

**Comment by vadim9999** [comment]
in 5-15 minutes I will push commit with MenuGenerator

**Comment by atherdon** [comment]
i have a question - where we get our data from? I mean what is the source, that give us data, that we later convert into a generated *.json file?

**Comment by vadim9999** [comment]
the source is folder data.

**Comment by atherdon** [comment]
hmm, will check it tomorrow deeply

**Comment by atherdon** [comment]
my goal with this task was to accomplish later this workflow:
Ex: we need to generate a file userGrocery. We grab all groceries, all users, and randomly assign few groceries to users and save it to our file. 
Ex: we have list of ingredients and list of users. We want randomly generate a list of favorite ingredients for users that we has.

does our current functionality cover that cases?

**Comment by vadim9999** [comment]
In https://github.com/GroceriStar/sd/pull/92

**Comment by vadim9999** [comment]
> Ex: we need to generate a file userGrocery. We grab all groceries, all users, and randomly assign few groceries to users and save it to our file.
>Ex: we have list of ingredients and list of users. We want randomly generate a list of favorite ingredients for users that we has.

Now it's just save data in file. I thought my task was to make function that will be writing data(object) in 
file

**Comment by atherdon** [comment]
yeah, you are right. our first task was to figure out how to save data to file. But I think now we need to move forward with it.

**Comment by vadim9999** [comment]
Do we have function that grab all groceries, all users, and randomly assign few groceries to users and save it to our file.Or it needs to create it?

**Comment by atherdon** [comment]
i don't know - you can go to fetch plugin and create it as a new issue - because we may need it soon. fetch is packed with new developers, so we don't need to code it now :)

**Comment by atherdon** [comment]
replying to your concerns - as first step - we can generate some of files, with data structures, that we have at some methods from graphQL.js project

**Comment by atherdon** [comment]
check this video https://www.loom.com/share/dbf7eb508a834038aba6db6a40f32cd6
